data/encounters/encounter.json

```diff
@@ -0,0 +1,9 @@
+{
+  "encounter":  {
```

Comment:
This JSON is only supposed to have encounter info, so there's no need to nest everything inside of an "encounter" field.